python/source/org/libreoffice/ide/eclipse/python/PythonProjectHandler.java

```diff
@@ -143,43 +153,32 @@ public void configureProject(UnoFactoryData pData, IProgressMonitor pMonitor) th
 
-        // The registration class name is always computed in the same way
-        //        String regclass = RegistrationHelper.getRegistrationClassName(unoprj);
-        //        unoprj.setProperty(P_REGISTRATION_CLASSNAME, regclass);
+        //Copy the Starting Python Source File under the source folder
+        IFolder sourceFolder = prj.getFolder(sourcesDir);
+        sourceFolder.create(true, true, pMonitor);
+        IPath sourcePath = sourceFolder.getProjectRelativePath();
+        Object[] args = { prj.getName() };
+        TemplatesHelper.copyTemplate(prj, "StartingPythonClass.py", PythonProjectHandler.class, sourcePath.toString(), //$NON-NLS-1$
+            false, args); //false denotes that the source .tpl filename and the destination filename are not same
 
-        //        // Java version
-        //        String javaversion = (String) pData.getProperty(PythonWizardPage.JAVA_VERSION);
-        //        unoprj.setProperty(P_JAVA_VERSION, javaversion);
+        // Refresh the project
+        try {
+            prj.refreshLocal(IResource.DEPTH_INFINITE, null);
+        } catch (Exception e) {
+        }
 
-        //        IProject project = ResourcesPlugin.getWorkspace().getRoot().getProject(unoprj.getName());
-        //
-        //        // Create the project structure
-        //        IJavaProject javaProject = JavaCore.create(project);
-        //        javaProject.open(pMonitor);
-        //
-        //        IPath sourcePath = unoprj.getFolder(unoprj.getSourcePath()).getFullPath();
-        //        IPath buildPath = unoprj.getFolder(unoprj.getBuildPath()).getFullPath();
-        //        System.out.println("SourcePath::" + sourcePath + "BuildPath::" + buildPath);
-        //
-        //        IClasspathEntry[] entries = new IClasspathEntry[] { JavaCore.newSourceEntry(sourcePath),
-        //            JavaRuntime.getDefaultJREContainerEntry(),
-        //            JavaCore.newLibraryEntry(buildPath, null, null, false) };
-        //
-        //        javaProject.setRawClasspath(entries, pMonitor);
-        //
-        //        // Add the registration files
-        //        RegistrationHelper.generateFiles(unoprj);
-        //
-        //        // Tests creation
-        //        Boolean usetests = (Boolean) pData.getProperty(PythonWizardPage.JAVA_TESTS);
-        //        if (usetests.booleanValue()) {
-        //            TestsHelper.writeTestClasses(unoprj);
-        //
-        //            IJavaProject javaprj = JavaCore.create(prj);
-        //            TestsHelper.addJUnitLibraries(javaprj);
-        //        }
+        // Show the newly created Python source file
+        IFolder srcDir = prj.getFolder(sourcePath);
+        IFile pythonSourceFile = srcDir.getFile(prj.getName() + ".py"); //$NON-NLS-1$
+        IWorkbenchWindow[] windows = PlatformUI.getWorkbench().getWorkbenchWindows();
+        WorkbenchHelper.showFile(pythonSourceFile,
+            windows[0].getActivePage());
```

Comment:
The source folder is being created and the empty skeleton file is put inside it.